ddl/db_integration_test.go

```diff
@@ -3036,21 +3036,6 @@ func TestAutoIncrementForceAutoIDCache(t *testing.T) {
 		return gid
 	}
 
-	// Rebase _tidb_row_id.
-	tk.MustExec("create table t (a int) AUTO_ID_CACHE 1")
-	tk.MustExec("alter table t force auto_increment = 2;")
-	tk.MustExec("insert into t values (1),(2);")
-	tk.MustQuery("select a, _tidb_rowid from t;").Check(testkit.Rows("1 1", "2 2"))
-	// Cannot set next global ID to 0.
-	tk.MustExec("alter table t force auto_increment = 0;")
-	tk.MustExec("alter table t force auto_increment = 1;")
-	require.Equal(t, uint64(3), getNextGlobalID())
-	// inserting new rows can overwrite the existing data.
-	tk.MustExec("insert into t values (3);")
-	tk.MustExec("insert into t values (3);")
-	tk.MustQuery("select a, _tidb_rowid from t;").Check(testkit.Rows("1 1", "2 2", "3 3", "3 4"))
-	tk.MustExec("drop table if exists t;")
-
```

Comment:
This code is duplicated with the following lines, it's a mistake when copying the code